src/ConfigEditor.tsx

```diff
@@ -10,13 +10,32 @@ interface State {}
 export class ConfigEditor extends PureComponent<Props, State> {
   render() {
     const { options, onOptionsChange } = this.props;
+    const onEnableImpersonationChange = (event: ChangeEvent<HTMLInputElement>) => {
+      onOptionsChange({...options, jsonData: {...options.jsonData, enableImpersonation: event.target.checked}})
+    }
     return (
       <div className="gf-form-group">
         <DataSourceHttpSettings
           defaultUrl="http://localhost:8080"
           dataSourceConfig={options}
           onChange={onOptionsChange}
         />
+
+        <h3 className="page-heading">Trino</h3>
+        <div className="gf-form-group">
+          <div className="gf-form-inline">
+            <InlineField
+              label="Impersonate logged in user"
+              tooltip="If enabled, set X-Trino-User to the current Grafana user"
```

Comment:
```suggestion
              tooltip="If enabled, set the Trino session user to the current Grafana user"
```

Reply:
Glad to hear that :)

I have applied your suggestion. Also I have signed the CLA, so I guess we'll have to wait for that to be processed.